queries.sql

```diff
@@ -159,4 +159,78 @@ FROM owners o
 LEFT JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY num_of_animals_owned DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+
+--- 1. Who was the last animal seen by William Tatcher?
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+JOIN vets vt ON v.vet_id = vt.id
+WHERE vt.name = 'William Tatcher'
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+--- 2. How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT a.id) AS num_of_animals_seen
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+JOIN vets vt ON v.vet_id = vt.id
+WHERE vt.name = 'Stephanie Mendez';

```

Comment:
- [OPTIONAL] Stephanie Mendez saw four animals and not 6 as shown in your result screenshot. kindly consider fixing it.